File: 02-greeter/greeter.erl
%% Simple Greeter program in Erlang
-module(greeter).
-author("M.V.Harish Kumar").
-compile([{date, "2023-12-07"}]).
-export([main/0]).

main() ->
    Name = string:trim(io:get_line("Please enter your name: ")),
    io:format("Hello, ~s! Hope you're fine~n", [Name]),
    erlang:halt(0).
    

File: 02-greeter/greeter.hs
-- Simple Greeter program in Haskell
-- Written by M.V.Harish Kumar on 11-07-2025
import System.IO (hFlush, stdout)

main = do
    putStr "Please enter your name: "
    hFlush stdout
    name <- getLine
    putStrLn $ "Hello, " ++ name ++ "! Hope you're fine"
